tests/test_config.py

```diff
@@ -158,9 +159,18 @@ def test_load_yaml_config_raises_error_if_unsafe_yaml():
     with open(YAML_PATH, "w") as fp:
         fp.write("hello: !!python/object/apply:os.system")
```

Comment:
It seems like this test didn't actually execute anything even with the unsafe loader. I need to dig in to figure out why